plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -632,6 +633,60 @@ string toVisible(MetaSpan.StatusColor s) {
     return format("status_%s", s);
 }
 
+/// DB data for coverage-visualization
+struct CoverageData{
+    bool status;
+    int begin;
+    int end;
+
+    static struct T {int value; bool status;}
+    T[] regions;
+
+    bool[int] lineList;
+
+    int byteCounter = 0;
+    int lineCounter = 1;
```

Comment:
I think, from looking at the code, that only `bool[int]` need to be members. the rest of the variables can be made local variables in method extractData?

I base it on what members are used in method lineExists.